internal/persistence/sql/migrations/templates/20210623162417_relationtuple.up.sql

```diff
@@ -0,0 +1,25 @@
+-- we can not fizz this migration because there is no check constraint support in fizz
+-- and it gets cumbersome with sqlite; having a working SQL version is actually way easier
+CREATE TABLE keto_relation_tuples
+(
+    shard_id                 UUID        NOT NULL,
+    network_id               UUID        NOT NULL,
```

Comment:
Just realized that I did not use `nid` because we also have a namespace ID and that is not descriptive enough to distinguish them... but maybe better to stay consistent still.